programs/server/Server.cpp

```diff
@@ -568,6 +568,8 @@ int Server::main(const std::vector<std::string> & /*args*/)
             if (config->has("zookeeper"))
                 global_context->reloadZooKeeperIfChanged(config);
 
+            global_context->reloadAuxiliaryZooKeepersConfigIfChanged(config);
```

Comment:
```
if (config->has("auxiliary_zookeepers"))
            global_context->reloadAuxiliaryZooKeepersConfigIfChanged(config);
```

Reply:
It's possible to remove `auxiliary_zookeepers` section to clean up those zookeeper connections